cmd/node/node.go

```diff
@@ -374,19 +374,20 @@ func (app *SpacemeshApp) Start(cmd *cobra.Command, args []string) {
 
 	app.unregisterOracle = func() { oracleClient.Unregister(true, pub.String()) }
 
-	//bo := oracle.NewBlockOracleFromClient(oracleClient, int(app.Config.CONSENSUS.NodesPerLayer))
 	nodesPerLayer := app.Config.CONSENSUS.NodesPerLayer
 	layersPerEpoch := app.Config.CONSENSUS.LayersPerEpoch
 	activationDb := &activation.ActivationDb{}      // TODO: initialize properly
```

Comment:
app.Config.CONSENSUS.NodesPerLayer is the size of the layer ? 
if so we already have a flag for that

Reply:
As Shaggy would put it: it wasn't me...
https://github.com/spacemeshos/go-spacemesh/commit/fb3af280e26293d1e2c86463798e6817dba1f8e5#diff-9341369a460f3f5c5336176dfbca5fbbR10

![shaggy](https://media1.tenor.com/images/f8927078793609393049c779e8210390/tenor.gif?itemid=4930192)

Reply:
my only point was that this sort of thing can take up lots of time if configured wrong, 
we have enough args as it is we don't need to add keep duplicates.
i opened #798